studio/lib/gotrue.ts

```diff
@@ -10,7 +10,10 @@ export const auth = new GoTrueClient({
 
 export const getAuthUser = async (token: String): Promise<any> => {
   try {
-    const { user, error } = await auth.api.getUser(token.replace('Bearer ', ''))
+    const {
+      data: { user },
+      error,
+    } = await auth.getUser(token.replace('Bearer ', ''))
```

Comment:
@alaister why do we need to pass `token` here if it's on the client side? `getUser()` should just use the current session right?